docs/data/base/components/input/OTPInput.js

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+import { Input as BaseInput } from '@mui/base/Input';
+import { Box, styled } from '@mui/system';
+
+function OTP({ seperator, inputCount, otp, setOtp }) {
```

Comment:
`otp` and `setOpt` are not really a standard way of providing a value. Could you change it to `value` and `onChange`?

Reply:
fixed here https://github.com/mui/material-ui/pull/40539/commits/7c6a2326f6972bf6cac5b06b9dd7d65a9a5b605b